lib/wanda/execution/evaluation.ex

```diff
@@ -114,11 +119,9 @@ defmodule Wanda.Execution.Evaluation do
       end)
       |> Enum.group_by(& &1.name)
       |> Enum.map(fn {name, expectation_evaluations} ->
-        %Catalog.Check{expectations: expectations} = Catalog.get_check(check_id)
```

Comment:
I understand that all of this code is simply to remove the need to run this `Catalog.get_check`, right?
This way all the checks execution flow depends on the coming parameters and it doesn't get anything during the flow.
Great

Reply:
Yes as `Catalog.get_check` reads a file now and in the future will probably retrieve checks from a persistence layer.